cms/templates/base.html

```diff
@@ -70,6 +71,7 @@
 
     <!-- view -->
     <div class="wrapper wrapper-view" dir="${static.dir_rtl()}">
+        <%static:optional_include_mako file="header_pre.html" is_theming_enabled="True" />
```

Comment:
Nit, but since the last upstream PR used hypens
(`body-post.html`) and we're using them above too 
(`head-post.html`), let's continue the trend here
(`header-pre.html`).